src/components/access-keys/AccessKeys.js

```diff
@@ -43,13 +87,16 @@ class AccessKeys extends Component {
             showSubOpen: 0,
             showSubData: null
         }))
+
+        this.handleConfirmClear()
     }
 
     handleDeauthorize = () => {
         const publicKey = this.state.showSubData.public_key
 
         this.setState(() => ({
-            loader: true
+            loader: true,
```

Comment:
any reason why 2 different loader states are needed? Shouldn't global loader state be enough here?

Reply:
Good point